docs/source/documentation.rst

```diff
@@ -125,6 +125,11 @@ AxonDeepSeg package with the following commands::
         cd axondeepseg
         git pull
         pip install -e .
+        
```

Comment:
I would include this in the blue box note to clarify that it is part of the "re-install".